script/hassfest/manifest.py

```diff
@@ -362,8 +364,39 @@ def validate_manifest(integration: Integration, core_components_dir: Path) -> No
         validate_version(integration)
 
 
+_SORT_KEYS = {"domain": ".domain", "name": ".name"}
+
+
+def _sort_manifest_keys(key: str) -> str:
+    return _SORT_KEYS.get(key, key)
+
+
+def sort_manifest(integration: Integration) -> bool:
+    """Sort manifest."""
+    keys = list(integration.manifest.keys())
```

Comment:
We can drop `.keys()`.

Reply:
I'll add a rule to our CI tomorrow that prevent this from ever happing again 😓